src/main/kotlin/com/openlattice/shuttle/hazelcast/processors/ReloadFlightEntryProcessor.kt

```diff
@@ -0,0 +1,37 @@
+package com.openlattice.shuttle.hazelcast.processors
+
+import com.dataloom.mappers.ObjectMappers
+import com.google.common.base.Preconditions.checkState
+import com.kryptnostic.rhizome.hazelcast.processors.AbstractRhizomeEntryProcessor
+import com.openlattice.shuttle.Flight
+import com.openlattice.shuttle.control.Integration
+import java.io.File
+
+class ReloadFlightEntryProcessor : AbstractRhizomeEntryProcessor<String, Integration, Integration>() {
+
+    companion object {
+        private const val serialVersionUID = -6602384557982348L
+        private val mapper = ObjectMappers.getYamlMapper()
+    }
+
+    override fun process(entry: MutableMap.MutableEntry<String, Integration>): Integration {
+        val integration = entry.value
+        val flightFilePath = integration.flightFilePath
+        checkState(flightFilePath != null, "Integration with name ${entry.key} does not have a flight file path")
+        val updatedFlight = mapper.readValue(File(integration.flightFilePath!!), Flight::class.java)
```

Comment:
Whoa, I have never thought of the implications of this but since EntryProcessors can run on different nodes, can't this potentially result in trying to read a file that definitely doesn't exist on a node?

Reply:
Ah. This sounds right to me. I may need to ask you further about this outside of github.

Reply:
Potential fix: flight yamls will be expected to be stored on github and instead of reading from a file we will read from a url (i.e. raw.githubusercontent.pathtoflight.yaml)